### PR TITLE
Fix for kickstart liveimg ported to f30/master

### PR DESCRIPTION
**Pull request by bcl**

### CODE REVIEW

File: pyanaconda/payload/livepayload.py [hunk 1]
```diff
@@ -169,17 +169,31 @@ def install(self):
...
+        # And kernel-install depends on it being set
+        util.execInSysroot("systemd-machine-id-setup", [])
+
         if os.path.exists(util.getSysroot() + "/usr/sbin/new-kernel-pkg"):
             use_nkp = True
         else:
             log.warning("new-kernel-pkg does not exist - grubby wasn't installed?")
             use_nkp = False
 
+            # Remove the bls entries included in the payload, they need to be generated to match the mountpoints
```
Comment (jkonecny12) [inline pyanaconda/payload/livepayload.py]: Please wrap also this line to 99 characters.

**Comment by jkonecny12** [comment]
Hi @martinezjavier ,
Could you please verify the last commit in this PR.
Thanks!

**Comment by bcl** [comment]
Ok, rewrote this with the suggestions and keeping adamw's method for creating the entries. The big difference being always making a new machine-id, and always removing any old BLS entries.

**Comment by bcl** [comment]
Rebased on current f30-devel. Note that I have not tested this lately.

**Comment by poncovka** [comment]
Jenkins, test this please.

**Comment by poncovka** [comment]
Tested live installation and automated installation with `liveimg` using image and tar.

**Pull request merged**